utils/land-patterns.stanza

```diff
@@ -375,6 +512,91 @@ public pcb-landpattern ipc-two-pin-landpattern (part-name:String, pol?:True|Fals
   make-ipc-two-pin-landpattern(part-name, true, pol?)
   ref-label()
 
+; Make a landpattern with pads along the four edges of the landpattern
+public defn make-four-sided-landpattern (
```

Comment:
A four sided landpattern computes the poses of its pads using the difference between inner/outer dimensions. The pitch is defined as `x(inner-dims) / pads-along-top` for top/bottom rows, and `y(inner-dims) / pads-along-side` for left/right columns.